OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -33,26 +34,337 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        </msItem>
+
+
+                        <msItem xml:id="ms_i5" corresp="#h1">
+                            <locus from="114v" to="118r"/>
+                            <title type="complete" ref="LIT3051RepCh331"
+                                ><!--Hymn to the Holy Trinity workfile no fidal no translit no translat--></title>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <locus target="#114v"/>
+                                <hi rend="rubric"> በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ በስመ፡</hi> ሥላሴ፡
+                                አአምን፡ ዘንተ፡ ሥላሴ፡ ዘምስለ፡ ሰብሖ፡ ለዘይፆረረኒ፡ ተመኪሆ፡ ዋለ፡ እግዚአብሔር፡ ይንጽሖ፡ በሰይፈ፡
+                                መለኮት፡ በሊህ፡ </incipit>
+                            <explicit xml:lang="gez"><locus target="#118r"/>፫ገጽ፡ ዘኢይትከፈል፡ ፩ራዕይ፡
+                                ወ፩አካል፡ ዕሩያነ፡ አምሳል፡ ብሩካነ፡ ቃል፡ እግዚአብሔር፡ ዋህድ፡ በአካሉ፡ ወምሉዕ፡ ዝምቅድመ፡ ዓለም፡
+                                    ህል<add place="inline">ው</add> እስከ፡ <sic>ለዓም፡</sic> ይሄሉ፡ ትጉሃነ፡
+                                ሰማይ፡ ኪያሁ፡ ይስእሉ፡ ቅዱስ፡ ቅዱስ፡ ቅዱስ፡ እግዚአብሔር፡ ጸባዖት፡ ይብሉ፡ ወስብሐቲሁ፡ ይመልዕ፡
+                                በኵሉ፡ ለገብርከ፡ <hi rend="rubric"><del rend="erasure" unit="lines"
+                                        extent="1"/></hi>
+                            </explicit>
+                        </msItem>
+
+                        <msItem xml:id="ms_i6" corresp="#h1">
+                            <locus from="118r" to="118v"/>
+                            <title type="complete" ref="LIT5962PrayerThreeNames"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <locus from="118r" to="118v"/>
+                                <hi rend="rubric"> በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ በ</hi> ስመ፡
+                                ፫አስማት፡ አአትብ፡<pb n="118v"/> ገጽየ፡ አአትብ፡ ድኅሬየ፡ በዘቦቱ፡ ተወኪልየ፡ ዕመውዕ፡ ፀርየ፡
+                                በስመ፡ ፫አስማት፡ አአትብ፡ የማንየ፡ በስመ፡ ፫አስማት፡ አአትብ፡ ፀጋምየ፡ በዘቦቱ፡ ተወኪልየ፡ ዕመውዕ፡
+                                ፀርየ፡ ወጸላዕትየ፡ በስመ፡ ፫አስማት፡ ገደፍኩ፡ ነፍስየ፡ ወሥጋየ፡ ወበስመ፡ ፫አስማት፡ ገደፍኩ፡ እስከ፡
+                                ርእስየ፡ በዘቦቱ፡ ዕመውዕ፡ ፀርየ፡ ለዓለመ፡ ዓለም፡ አሜን። ኦእግዚኦ፡ ዕቀበኒ፡ ለገብርከ፡ <hi
+                                    rend="rubric"><del rend="erasure" unit="lines" extent="1"/></hi>
+                                <persName ref="PRS13017GabraMar"><add place="below">ገበረ፡ ማርያም።
+                                    </add></persName>
+                            </incipit>
+                        </msItem>
+
+
+
                     </msContents>
-                    
+
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>A hole on <locus target="#54"/>
```

Comment:
Should there be a full stop after such statements? I think I usually would have one there.